src/dec_text.c

```diff
@@ -1508,6 +1508,9 @@ text_parse_index(io_read_proxy_t *fd, as_vector *ns_vec, uint32_t *line_no,
 	as_vector_init(&index->path_vec, sizeof (path_param), 25);
 	path_param path;
 
+	// This loop parses n_path path specs from sindex entries in the
+	// backup file. These include the space after n_paths to the path type.
+	// The code supports multiple path specs, but currently there is only ever one.
```

Comment:
```suggestion
	// This loop parses n_path path specs from sindex entries in the
	// backup file. These include the space after n_paths to the path type.
	// The code supports multiple path specs, but there is only ever one.
	// paths should be changed to a single value rather than a vector to
	// avoid future confusion.
```